src/test/compile-fail/fn-item-type.rs

```diff
@@ -24,20 +24,20 @@ fn main() {
     //~^ ERROR mismatched types
     //~|  expected type `fn(isize) -> isize {foo::<u8>}`
     //~|  found type `fn(isize) -> isize {bar::<u8>}`
-    //~|  expected fn item, found a different fn item
+    //~|  expected fn item
```

Comment:
Yeah, changes like this in particular are what I was worried about.  If we remove the `found` part the underline becomes very confusing.